### DISCUSSION TITLE
Community call

### DISCUSSION THREAD

**Discussion by MarcoGorelli** [narwhals-dev/narwhals#3540] [Announcements]
We'll have our first community call tomorrow, at 14:30 UTC

- Here's the link: meet.google.com/gcn-hbnx-imn
- Shared calendar: https://calendar.google.com/calendar/embed?src=27ff6dc5f598c1d94c1f6e627a1aaae680e2fac88f848bda1f2c7946ae74d5ab%40group.calendar.google.com&ctz=Europe%2FLondon

All welcome to join!

To find out what time that is for you, you can run:
```python
import polars as pl
from datetime import datetime

MY_TIME_ZONE = 'Europe/London'  # replace with your time zone
print(pl.Series([datetime(2020, 6, 7, 14, 30)], dtype=pl.Datetime('us', 'UTC')).dt.convert_time_zone(MY_TIME_ZONE))
```

**Comment by anopsy** [comment]
See you at:
shape: (1,)
Series: '' [datetime[μs, Europe/Amsterdam]]
[
	2020-06-07 16:30:00 CEST
]

**Comment by dawnwages** [comment]
Hi hi ! Are you still having community calls?

**Comment by MarcoGorelli** [comment]
hey @dawnwages ! yup, the next one's on the 7th of October, would be great to have you join!

**Comment by FBruzzesi** [comment]
Hey @dawnwages 👋🏼 Yes absolutely! We are having them once a month. The [calendar](https://calendar.google.com/calendar/embed?src=27ff6dc5f598c1d94c1f6e627a1aaae680e2fac88f848bda1f2c7946ae74d5ab%40group.calendar.google.com&ctz=Europe%2FLondon) is up to date (next one is on Nov 7th).

We also have a [discord](https://discord.gg/V3PqtB4VA4) for async conversations
